### PR TITLE
WIP - Resources Page

### LINKED ISSUE #4: [On-Going] Resources Page
Many new people to coding, don't know where to look for resources on what they are learning. 

~~We will need to create a page~~ We have a page for posting links to resources that students can use to learn. If you see a good resource that isn't on the page, feel free to make a PR to add it!

### PR DESCRIPTION
**Pull request by mattstub**
This resolves #4, however I have it listed out as a WIP because I didn't know if there were a handful of other links that we wanted to post at this time or if there was another page structure that we possibly wanted to use.

### CODE REVIEW

File: docs/resources.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                            <li><a href="https://www.w3schools.com/" target="_blank" title="W3Schools" rel="external">W3Schools</a> - A vast amount of information and tutorials that you can read and try for free of varying front-end development technologies. W3Schools offers the ability to test your skills and earn certifications at a cost, but still offers a tremendous amount of free information.</li>
+                        </ul>
+                    <h2 id="tutorials">Tutorials & Courses</h2>
+                        <ul>
+                            <li><a href="https://www.udemy.com/the-complete-javascript-course/" target="_blank" title="The Complete JavaScript Course" rel="external">The Complete JavaScript Course</a> by Jonas Schmedtmann - This course offered by Udemy will cost $10 but does a great job of explaining the basics of JS and how it really works at the core. If you are looking for some help with DOM traversal this is a great resource. There are also a list of wonderful resources he has compiled on his personal webpage, <a href="http://codingheroes.io/" target="_blank" title="CodingHeroes.io" rel="external">CodingHeroes.io</a></li>
```
Comment (CarsonMcKinstry) [inline docs/resources.html]: This doesn't always cost $10, so be careful on the wording here.

File: docs/resources.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                            <li><a href="https://www.sololearn.com/" target="_blank" title="SoloLearn" rel="external">SoloLearn</a> - Great little resource if you are looking for help with core programming fundamentals. SoloLearn offers fundamental tutorials on plenty of languages!</li>
+                            <li><a href="https://www.udemy.com/courses/development/" target="_blank" title="Udemy" rel="external">Udemy</a> - A vast library of learning resources for all kinds of topics. This link will take you to the development side, which offers thousands of courses for every programming language and topic you could imagine.</li>
+                        </ul>
+                    <h2 id="books">Books</h2>
+                        <ul>
+                            <li><a href="http://eloquentjavascript.net/" target="_blank" title="Eloquent JS" rel="external">Eloquent JS</a> - You can find a link to the paperback version as well here, or just utilize the Electronic versions. Some taught Eloquent JS to be the be-all-end-all reference for Vanilla JS</li>
```
Comment (CarsonMcKinstry) [inline docs/resources.html]: Working on finding out if Techlahoma has an Amazon Affiliate link

**Review by CarsonMcKinstry — COMMENTED** [review]
Take a look at the comments. 

Also, the page seems a little bit wordy.

File: docs/resources.html (hunk 1 above)
Comment (mattstub) [reply docs/resources.html]: 10-4 let me make some adjustments to the wording over all to also clean up the wordiness and will address this as well.

**Comment by mattstub** [comment]
Going to pull this back and push to a branch so i can work on something while I wait on the resources page

**Pull request closed without merging**